Sources/TuistAutomation/Device/DeviceController.swift

```diff
@@ -0,0 +1,168 @@
+import Command
+import FileSystem
+import Foundation
+import Mockable
+import Path
+import TuistSupport
+import XcodeGraph
+
+enum DeviceControllerError: FatalError {
```

Comment:
Every time I see `FatalError` it makes me think we should iterate on the pattern to embrace plain `Error`, which we can also raise from external packages like FileHandler owned by Tuist.

Reply:
I agree, it might be worth refactoring, but until then, let's keep things consistent